### DISCUSSION TITLE
Celescope for snRNAseq annotation

### DISCUSSION THREAD

**Discussion by amarinderthind** [singleron-RD/CeleScope#283] [General]
Wondering if Celescope has any specific feature for Gene count from snRNAseq data, where intronic regions must be included in the annotation for each gene to account for the increased abundance of unspliced transcripts? [https://bioconductor.org/books/3.18/OSCA.advanced/single-nuclei-rna-seq-processing.html](https://bioconductor.org/books/3.18/OSCA.advanced/single-nuclei-rna-seq-processing.html )

**Comment by zhouyiqi91** [comment]
celescope uses `GeneFull_Ex50pAS` by default which already include introns for quantification. It works for both scRNA and snRNA.
https://github.com/alexdobin/STAR/issues/1460
